code/modules/shuttle/on_move.dm

```diff
@@ -318,8 +318,11 @@ All ShuttleMove procs go here
 	. = ..()
 
 	var/knockdown = movement_force["KNOCKDOWN"]
-	if(knockdown)
+	//austation begin -- cat tails prevent shuttle knockdown
+	var/obj/item/organ/tail/cat/O = getorganslot(ORGAN_SLOT_TAIL)
+	if(knockdown && !istype(O))
 		Paralyze(knockdown)
+	//austation end
```

Comment:
make this work for lizard tails too, considering they're far thicker and stronger than cat tails, it makes sense for them to get this buff too

Reply:
(implying shuttle knockdown for 0.5 seconds means anything)